plugins/instruct-lab/src/fms_acceleration_ilab/framework_plugin_padding_free.py

```diff
@@ -0,0 +1,153 @@
+# Copyright The FMS HF Tuning Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Standard
+from typing import Dict, Tuple
+from packaging import version
+import warnings
+
+# Third Party
+from fms_acceleration import AccelerationPlugin
+from peft import LoraConfig
+from transformers import TrainingArguments, __version__ as transformers_version, DataCollatorForSeq2Seq
+from accelerate import Accelerator
+import torch
+from types import MethodType
+from torch.utils.data import DataLoader
+
+SUPPORTED_TRANSFORMERS_VERSION = "4.43"
```

Comment:
call this `TRANSFORMERS_VERSION` and put comment to explain that this was the version that padding free was merged in.